nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/GlobalConfigDescriptor.java

```diff
@@ -30,82 +31,87 @@
 
 /**
  * XMap Descriptor for the {@link org.nuxeo.ecm.core.convert.api.ConversionService} configuration.
- *
- * @author tiry
  */
 @XObject("configuration")
 public class GlobalConfigDescriptor implements Serializable {
 
+    private static final long serialVersionUID = 1L;
+
+    public static final boolean DEFAULT_CACHE_ENABLED = true;
+
     public static final long DEFAULT_GC_INTERVAL_IN_MIN = 10;
 
     public static final int DEFAULT_DISK_CACHE_IN_KB = 10 * 1024;
 
-    private static final String CACHING_DIRECTORY = "convertcache";
+    public static final String DEFAULT_CACHING_DIRECTORY = "convertcache";
 
-    private static final long serialVersionUID = 1L;
+    @XNode("enableCache")
+    protected Boolean enableCache;
 
-    protected long GCInterval = DEFAULT_GC_INTERVAL_IN_MIN;
+    public boolean isCacheEnabled() {
+        return enableCache == null ? DEFAULT_CACHE_ENABLED : enableCache.booleanValue();
+    }
 
-    protected int diskCacheSize = DEFAULT_DISK_CACHE_IN_KB;
+    @XNode("cachingDirectory")
+    protected String cachingDirectory;
 
-    @XNode("enableCache")
-    protected boolean enableCache = true;
+    public String getCachingDirectory() {
+        return cachingDirectory == null ? getDefaultCachingDirectory() : cachingDirectory;
+    }
 
-    @XNode("cachingDirectory")
-    protected String cachingDirectory = defaultCachingDirectory().getAbsolutePath();
+    protected String getDefaultCachingDirectory() {
+        File cache = new File(Environment.getDefault().getData(), DEFAULT_CACHING_DIRECTORY);
+        return cache.getAbsolutePath();
+    }
 
-    public long getGCInterval() {
-        return GCInterval;
+    /** @since 9.1 */
+    public void clearCachingDirectory() {
+        File cache = new File(getCachingDirectory());
+        if (cache.exists()) {
+            try {
+                FileUtils.deleteDirectory(cache);
+            } catch (IOException e) {
+                throw new RuntimeException("Cannot create cache dir " + cache, e);
```

Comment:
That was the old code but you're right it's cleaner. I pushed an updated version.